src/client/dfuse/pil4dfs/int_dfs.c

```diff
@@ -1220,20 +1114,23 @@ query_path(const char *szInput, int *is_target_path, dfs_obj_t **parent, char *i
 			/* the item under root directory */
 			if (pos == (*dfs_mt)->len_fs_root) {
 				(*parent_dir)[0] = '/';
-				path_len         = 1;
+				parent_dir_len   = 1;
 			} else {
+				char *parent_path;
+
+				full_path_parse[pos] = '\0';
+				parent_path          = full_path_parse + (*dfs_mt)->len_fs_root;
+				/* path_dir_len is length of the string, without termination */
```

Comment:
"path_dir_len" in comment should be updated as "parent_dir_len".

Reply:
> * [ ]  Fix comment error

Fixed with commit 9eb019038